discord/ext/commands/errors.py

```diff
@@ -521,19 +521,18 @@ def __init__(self, name, original):
         super().__init__(fmt.format(name, original), name=name)
 
 class ExtensionNotFound(ExtensionError):
-    """An exception raised when an extension failed to be imported.
+    """An exception raised when an extension is not found.
 
     This inherits from :exc:`ExtensionError`
 
     Attributes
     -----------
     name: :class:`str`
         The extension that had the error.
-    original: :exc:`ImportError`
-        The original exception that was raised. You can also get this via
-        the ``__cause__`` attribute.
+    original: :class:`NoneType`
+        Always ``None`` for backwards compatibility.
     """
-    def __init__(self, name, original):
-        self.original = original
+    def __init__(self, name):
```

Comment:
```suggestion
    def __init__(self, name, original=None):
```

I've given this some thought and I think to absolutely minimize breaking changes even further the parameter count should stay the same with a default and then subsequently ignored.